src/main/resources/openshift/deploy.sh

```diff
@@ -2,7 +2,9 @@
 
 set -e
 
-OCP_PROJECT=rhamt
+if [ -z "$OCP_PROJECT" ]; then
```

Comment:
this will not work correctly if you use different name.
the non rhamt project doesn't redirect to rhamt-web-console URL and shows EAP/Wildfly root page with Welcome to JBoss EAP 7. Not sure why is not redirect added, in logs there is no error or warning about failing to add redirect from `eap.cli`.

https://github.com/jsight/windup-web-distribution/blob/WINDUP-1555/src/main/resources/openshift/README.md is not mentioning the setting of property only editing `deploy.sh` script property